### DISCUSSION TITLE
Add A Way to Disable a immersive portal without disabling the portal.

### DISCUSSION THREAD

**Discussion by suace69** [iPortalTeam/ImmersivePortalsMod#1248] [Ideas]
Or Add Create(Mod), Compatibility Please.

**Comment by qouteall** [comment]
What do you mean by "Add A Way to Disable a immersive portal without disabling the portal"?

**Reply by suace69** [reply]
What I mean is a way to convert an immersive portal to a vanilla one without having to disable the mod light the portal and enable the mod again.

**Reply by qouteall** [reply]
Unfortunately this mod does not have such functionality.
